app/data_model/database.py

```diff
@@ -79,7 +79,8 @@ def _create_session_and_engine_with_retry():
     for i in range(settings.EQ_SERVER_SIDE_STORAGE_DATABASE_SETUP_RETRY_COUNT):
         try:
             return _create_session_and_engine()
-        except Exception as e:
+        except Exception as e:  # pylint: disable=broad-except
```

Comment:
Hard to predict what can go wrong when trying to setup the database; so this retries x times before giving up and throwing the last exception caught.